src/main/scala/com/test/MainTask.scala

```diff
@@ -1,23 +1,21 @@
 package com.test
```

Comment:
Just remove this file. CE3 support in Monix is already long overdue. And IMO it should not be promoted for new projects. Having it as an example for legacy version in git history is enough.